strings/ngram.py

```diff
@@ -0,0 +1,29 @@
+"""
+https://en.wikipedia.org/wiki/N-gram
+
+
+"""
+
+
+def create_ngram(sentence: str, ngram_size: int) -> list:
+    """
+    Create ngrams from a sentence
+    :param sentence: str
+    :param ngram_size: int
+    :return: list
```

Comment:
If these lines only repeat information that has already been stated in the type hints then they are not needed.
```suggestion
```